modules/ROOT/pages/Development/BeginnersGuide/dependencies.adoc

```diff
@@ -319,8 +340,9 @@ from the Satisfactory Modding GitHub.
 
 [TIP]
 ====
-If you're planning to clone the project,
-skip to the link:#_satisfactory_mod_manager[next heading].
+If you're planning to use Git to clone the project,
+skip this section and move to the xref:Development/BeginnersGuide/project_setup.adoc[next page]
```

Comment:
I would prefer to link to "Next Steps" on the same page, as it would avoid accidentally jumping over any steps on the same page and seems easier to maintain (it's a link to the same page).

Reply:
I specifically rearranged the page so there would be no further next steps on the same page; I think a heading link that doesn't scroll you anywhere would be more confusing

Reply:
I think linking to the next page is fine here. The link is at the end of the page, so more people would be confused as to why the link doesn't lead anywhere, as if it is broken.